.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 vendor
+config/phpstan/phpstan.neon
```

Comment:
Warum das?

Reply:
config/phpstan/phpstan.neon wird generiert, config/phpstan/phpstan.neon.dist ist die default-config in die dann includes und die excludes_paths rein kommen
Oder meinst Du ist eh egal, weil es eh dev ist? Ist trotzdem denke ich besser, bevor jemand aus versehen commitet.